### PR TITLE
#2 Configure the React app environment 

### LINKED ISSUE #2: Configure the react app environment

### PR DESCRIPTION
**Pull request by drfidel**
## Description
1.Setup project for REACT environment which fixes Issue #2.
2.Setup environment by running npm install in console in app directory.
 
List any dependencies that are required for this change.
-react, react-dom, prop-tools, babel, eslint and setup using parcel

Fixes #(issue number 2)

## How Has This Been Tested
No test written

## Checklist:

- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added necessary inline code documentation
- [ ] I have added tests that prove my fix is effective and that this feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "presets": ["@babel/preset-react"]
+}
```
Comment (mariamiah) [inline .babelrc]: add a new line here

**Comment by mariamiah** [comment]
We should also have either yarn.lock or package-lock.json but not both...check the package manager used

**Comment by mariamiah** [comment]
Please edit the pull request description above to state the issue being fixed, what is being setup and how to test the branch

**Comment by AineKiraboMbabazi** [comment]
I will not be contributing much on this because I don't do react.

**Review by mariamiah — APPROVED** [review]
LGTM

**Pull request merged**